client/hostinfo_unix.cpp

```diff
@@ -73,12 +78,16 @@
 
 #include <sys/stat.h>
 
+#ifdef ANDROID
+#include <sys/wait.h>
+#endif
+
 #if HAVE_SYS_SWAP_H
-#if defined(ANDROID) && !defined(ANDROID_64)
-#include <linux/swap.h>
-#else
+// #if defined(ANDROID) && !defined(ANDROID_64)
+// #include <linux/swap.h>
+// #else
```

Comment:
I guess this is part of `../android/android_config.h` now? If not please remove it completely or add a comment why it was commented out but left in.

Reply:
Yes, this is not needed anymore. I'll remove it. Thanks